modules/space/translation/horizonstranslation.cpp

```diff
@@ -35,22 +35,24 @@
 #include <ghoul/lua/lua_helper.h>
 #include <filesystem>
 #include <fstream>
+#include <type_traits>
 
 namespace {
     constexpr const char* _loggerCat = "HorizonsTranslation";
+    constexpr int8_t CurrentCacheVersion = 2;
 } // namespace
 
 namespace {
     constexpr openspace::properties::Property::PropertyInfo HorizonsTextFileInfo = {
         "HorizonsTextFile",
         "Horizons Text File",
-        "This value is the path to the text file generated by Horizons with observer "
-        "range and Galactiv longitude and latitude for different timestamps."
+        "This value is the path to the file or files generated by Horizons with "
+        "either a Vector table or an Observer table with the correct settings."
```

Comment:
I wrote about them in the [header file](https://github.com/OpenSpace/OpenSpace/blob/af0a6d61cb4c828277f916dd9a33b6188d980724/modules/space/translation/horizonstranslation.h#L47) and will probably edit our [wiki page](http://wiki.openspaceproject.com/docs/builders/ephemeris/horizons) to use the new version as well. Since the text was so long I didn't want to repeat it here but I can link to the wiki when it has been updated.

Reply:
👍 Adding "(see wiki)", and not the full link, would probably be enough. Links might break in the future and would be nice to not have to maintain it